front50-s3/src/main/java/com/netflix/spinnaker/front50/config/S3Config.java

```diff
@@ -112,7 +112,7 @@ public ApplicationPermissionDAO applicationPermissionDAO(StorageService storageS
 
   @Bean
   public ServiceAccountDAO serviceAccountDAO(StorageService storageService, Registry registry) {
-    return new DefaultServiceAccountDAO(storageService, Schedulers.from(Executors.newFixedThreadPool(20)), 30000, registry);
+    return new DefaultServiceAccountDAO(storageService, Schedulers.from(Executors.newFixedThreadPool(5)), 30000, registry);
```

Comment:
lowered the size of these thread pools as it's unlikely we'll have sufficient service account documents that we need 20 threads to load in parallel ... ditto for snapshots (which may not even be fully supported in AWS).